doc/classes/AudioEffectNotchFilter.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectNotchFilter" inherits="AudioEffectFilter" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a Notch Filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect attenuates the frequences inside a narrow band around the [code]Cutoff HZ[/code], and cuts frequences out of this range.
```

Comment:
- Remove the comma before "and"
- "Attenuates the frequencies in a narrow band around the [member cutoff_hz]"